app/javascript/packs/details_pg.js

```diff
@@ -0,0 +1,13 @@
+function onReady() {
+    let cardToggles = document.getElementsByClassName('card-toggle');
+	for (let i = 0; i < cardToggles.length; i++) {
+		cardToggles[i].addEventListener('click', e => {
+			e.currentTarget.parentElement.parentElement.childNodes[3].classList.toggle('is-hidden');
```

Comment:
Something weird is going on with the indentation in this file.

Reply:
resolved